airflow/utils/db.py

```diff
@@ -1888,12 +1914,144 @@ def get_query_count(query_stmt: Select, *, session: Session) -> int:
     return session.scalar(count_stmt)
 
 
+def get_query_exists(query_stmt: Select, *, session: Session) -> bool:
```

Comment:
nitpick: I can't understand the meaning of this function at first glance. 👀 Not sure whether `check_query_exists` make more sense

Reply:
I was following the naming of `get_query_count` (admittedly not a very good one either). Using `check` instead makes sense.